alf/examples/her_target_navigation_states_conf.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) 2022 Horizon Robotics and ALF Contributors. All Rights Reserved.
```

Comment:
No.  See note at line #15.  Do you prefer I delete this?

Reply:
> No. See note at line #15. Do you prefer I delete this?

Yeah I think it's best to keep it as a local copy in case that someone complains the file cannot run after being checked in.

Reply:
Done.